backend/src/org/commcare/session/SessionDescriptorUtil.java

```diff
@@ -16,7 +16,8 @@ public static void loadSessionFromDescriptor(String sessionDescriptor,
             if (action.equals(SessionFrame.STATE_COMMAND_ID)) {
                 session.setCommand(tokenStream[++current]);
             } else if (action.equals(SessionFrame.STATE_DATUM_VAL) ||
-                    action.equals(SessionFrame.STATE_DATUM_COMPUTED)) {
```

Comment:
We want to allow STATE_UNKNOWN type actions to be instantiated from the descriptor string in the same way that computed and datums are